apple-pie/index.html

```diff
@@ -4,18 +4,79 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! Ingredients 1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter 3 tablespoons
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
-    C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. Place the bottom crust in your pan. Fill with apples, mounded
-    slightly. Cover with a lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-    that it does not run off. Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-    degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+    <!-- Image -->
+    <div class="image-container">
+      <div class="pie-image">
+        <img src="/apple-pie/images/apple-pie.jpg" alt="" />
+      </div>
+      <div class="top-text-container">
+        <h1>Apple Pie</h1>
+      </div>
+    </div>
+    <!-- How to / Content Section -->
+    <div class="page-outer-container">
+      <div class="page-inner-container">
+        <section class="howto">
+          <h4>
+            This was my grandmother's apple pie recipe. I have never seen
+            another one quite like it. It will always be my favorite and has won
+            me several first place prizes in local competitions. I hope it
+            becomes one of your favorites as well!
+          </h4>
+          <img
+            id="recipe-info"
+            src="/apple-pie/images/recipe-info.png"
+            alt=""
+          />
+          <div class="hr"></div>
+
+          <h3>Ingredients</h3>
+          <ul>
+            <li>1 recipe pastry for a 9 inch double crust pie</li>
+            <li>1/2 cup unsalted butter</li>
+            <li>3 tablespoons all-purpose flour</li>
+            <li>1/4 cup water</li>
+            <li>3 tablespoons all-purpose flour</li>
+            <li>1/4 cup water</li>
+            <li>1/2 cup white sugar</li>
+            <li>1/2 cup packed brown sugar</li>
+            <li>8 Granny Smith apples - peeled, cored and sliced</li>
+          </ul>
+        </section>
+
+        <section class="howto">
+          <img
+            id="recipe-info"
+            src="/apple-pie/images/cooking-info.png"
```

Comment:
depending on which part of a page (which URL you are on) the / before file srcs might not be what you want. 
have a look at this: 
https://webmasters.stackexchange.com/questions/56840/what-is-the-purpose-of-leading-slash-in-html-urls
tldr: it might break when you are nested in sites and your image src is not nested :+1: